src/examples/sumologic-unmounted-pvc-deleter-cronjob.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: sumologic-unmounted-pvc-deleter
```

`PVC Cleaner` rolls off the tongue better to me ;)
```suggestion
  name: sumologic-pvc-cleaner
```